api/tests/test_faq_rebuild_optimization.py

```
@@ -0,0 +1,283 @@
+
+@pytest.mark.unit
+def test_bulk_delete_only_verified_triggers_rebuild(faq_service_with_mock_callback):
+    """Test that bulk deleting only verified FAQs triggers rebuild."""
+    faq_service, mock_callback = faq_service_with_mock_callback
+
+    # Get all verified FAQs
+    all_faqs = faq_service.get_all_faqs()
+    verified_faq_ids = [faq.id for faq in all_faqs if faq.verified]
+
+    # Delete them all
+    success, failed, failed_ids = faq_service.bulk_delete_faqs(verified_faq_ids)
```

Ruff (0.14.3)

Unpacked variable `failed_ids` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)